build/build.xml

```diff
@@ -814,7 +814,7 @@
 				<include name="accessControl-${status}-*.zap"/>
 				<include name="cspscanner-${status}-*.zap"/>
 				<include name="formhandler-${status}-*.zap"/>
-				<include name="jxbrowser*-{status}-*.zap"/>
```

Comment:
Ah, it was missing the `$` in the variable, but new regex works too (the status is not really important).